### PR TITLE
Fix Roslyn wiki links

### PR DESCRIPTION
**Pull request by johnallers**
Links to the Roslyn GitHub wiki are broken since it has been moved to the docs folder (see https://github.com/dotnet/roslyn/pull/41157).

### CODE REVIEW

**Comment by ktoliver** [comment]
Hi @ghogen @Mikejo5000 Could one of you take a look at this PR? Thanks!

File: docs/debugger/debugging-linq.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ End Function
  The revised query calls the function `IsEven` on each pass through the `items`. You can use the debugger windows to see whether each item meets the specified condition, and you can step through the code in `IsEven`. The predicate in this example is fairly simple. However, if you have a more difficult predicate you have to debug, this technique can be very useful.
 
 ## <a name="BKMK_EditandContinueNotSupportedforLINQ"></a> Edit and Continue Not Supported for LINQ
- Edit and Continue supports changes to LINQ queries with limitations. For details, see [EnC Supported Changes](https://github.com/dotnet/roslyn/wiki/EnC-Supported-Edits))
+ Edit and Continue supports changes to LINQ queries with limitations. For details, see [EnC Supported Changes](https://github.com/dotnet/roslyn/blob/master/docs/wiki/EnC-Supported-Edits.md))
```
Comment (Mikejo5000) [inline docs/debugger/debugging-linq.md:102]:
```suggestion
 Edit and Continue supports changes to LINQ queries with limitations. For details, see [EnC Supported Changes](https://github.com/dotnet/roslyn/blob/master/docs/wiki/EnC-Supported-Edits.md)
```

**Comment by PRMerger8** [comment]
@Mikejo5000 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by Mikejo5000 — APPROVED** [review]
Thanks for the fixes! This looks good.

**Comment by Mikejo5000** [comment]
#sign-off

**Pull request merged**